### PR TITLE
Fix status command

### PR DESCRIPTION
**Pull request by tomdee**
And allow users to choose their own image

### CODE REVIEW

File: calicoctl/calico_ctl/node.py [hunk 1]
```diff
@@ -436,6 +436,10 @@ def _start_node_container_rkt(ip, ip6, node_image, etcd_envs,
     files to mount on the container
     :return: None.
     """
+    if node_image == "calico/node:latest":
```
Comment (caseydavenport) [inline calicoctl/calico_ctl/node.py]: What about `calico/node:vX.X.X`? Don't we still want to pull those from docker?

Will the release script update this to be the correct version on release?

**Comment by caseydavenport** [comment]
One comment.

File: calicoctl/calico_ctl/node.py (hunk 1 above)
Comment (tomdee) [inline calicoctl/calico_ctl/node.py]: I _think_ it will. I'll be sure to check when I do the release.

**Comment by caseydavenport** [comment]
LGTM

**Pull request merged**